symengine/tests/logic/test_logic.cpp

```diff
@@ -137,7 +137,7 @@ TEST_CASE("And, Or : Basic", "[basic]")
     auto int1 = interval(integer(1), integer(2), false, false);
     auto int2 = interval(integer(1), integer(5), false, false);
     auto c1 = contains(x, int1);
-    auto c2 = contains(x, int2);
+    auto c2 = contains(symbol("y"), int2);
```

Comment:
I will revert this. I earlier changed this because initially I simplified `contains`.